tests/installation/addon_products_sle.pm

```diff
@@ -183,11 +176,7 @@ sub run {
             }
         }
     }
-    test_addonurl
-      if is_sle('>=15')
-      and !check_var('SCC_REGISTER', 'installation')
-      and (get_var('ALL_MODULES') || get_var('WORKAROUND_MODULES'));
-
+    test_addonurl if is_sle('>=15');
```

Comment:
I think here we shouldn't let migration test enter the function test_addonurl.  Maybe we can use is_upgrade, but I'm not very sure your purpose so this just a suggestion.  Besides, please verify it works by clone following jobs:
https://openqa.suse.de/tests/3360463#step/addon_products_sle/4
https://openqa.suse.de/tests/3360463#step/addon_products_sle/4

Reply:
cloned to https://openqa.suse.de/tests/3362813#. waiting for the results

Reply:
Yes, I agree to use ADDONURL as condition for this function since migration test on s390x use this variable also.